clowdapp.yaml

```diff
@@ -23,8 +23,7 @@ objects:
     - host-inventory
     - ingress
     - rbac
-#  blocked https://issues.redhat.com/browse/RHCLOUD-10212
-#    - remediations
+    - remediations
```

Comment:
I wonder if we should add this as an optionalDependency. Our import SSG job gracefully fails when the remediations service is not available, notwithstanding the open SSL issue we found yesterday.

Reply:
I agree, updated as an optional dependency :+1: